plugin.yaml

```diff
@@ -207,6 +223,17 @@ relationships:
 #                 - establish: cloudstack.neutron_plugin.router.connect_subnet
 #                 - unlink: cloudstack.neutron_plugin.router.disconnect_subnet
 
+    cloudify.cloudstack.volume_contained_in_vm:
+            derived_from: cloudify.relationships.contained_in
+            source_interfaces:
+                cloudify.interfaces.relationship_lifecycle:
+                    establish:
+                        implementation: cloudstack.cloudstack_plugin.volume.attach_volume
```

Comment:
You can map the operations this way:
establish: cloudstack.cloudstack_plugin.volume.attach_volume
etc...